test/FedToken.js

```diff
@@ -0,0 +1,63 @@
+
+// Hardhat tests are normally written with Mocha and Chai.
+
+// We import Chai to use its asserting functions here.
+const { expect } = require("chai");
+const { ethers } = require("hardhat")
+
+describe("FedToken contract", function () {
+  // Mocha has four functions that let you hook into the the test runner's
+  // lifecyle. These are: `before`, `beforeEach`, `after`, `afterEach`.
+
+  // They're very useful to setup the environment for tests, and to clean it
+  // up after they run.
+
+  // A common pattern is to declare some variables, and assign them in the
+  // `before` and `beforeEach` callbacks.
+
+  let Token;
```

Comment:
nit: you can delete lines 18, 21, 22 & 23 if you're not using them

Reply:
sounds good